App.css

```diff
@@ -1,3 +1,7 @@
+/* Copyright 2023 Adobe. All Rights Reserved.
+NOTICE: Adobe permits you to use, modify, and distribute this file in
```

Comment:
Misspelled in all files; do a find/replace

Reply:
My eyes were playing tricks on me. Sorry.

Reply:
No worries! I got side-tracked and only just now made it back to this PR. 👍